packages/gatsby-theme-docs/src/overrides/site-icon.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import SiteIcon from '../icons/generated/OpenSourceSmall';
+
+// A React component to be rendered next to the microsite title
+// eslint-disable-next-line react/display-name
+export default () => {
+  return (
+    <div>
```

Comment:
unncessary div wrapper.  If a specific icon needs it it can do that in its override